xwiki-platform-core/xwiki-platform-sharepage/xwiki-platform-sharepage-ui/src/main/resources/XWiki/SharePage.xml

```diff
@@ -337,7 +337,7 @@ $!{escapetool.xml($message)}
 &lt;/pre&gt;
 #end
 
-&lt;p&gt;To visit this document online, open &lt;a href="${doc.getExternalURL()}"&gt;${doc.getExternalURL()}&lt;/a&gt; in your browser.&lt;/p&gt;
+&lt;p&gt;To visit this document online, open &lt;a href="${doc.getExternalURL()}"&gt;${util.decodeURI("${doc.getExternalURL()}")}&lt;/a&gt; in your browser.&lt;/p&gt;
```

Comment:
hmm I wonder if that's correct. Imagine that we have a page named "my/Page". It would be encoded as "http://.../my%2FPage/..." but if you call util.decodeURI on it, you'll get: "http://.../my/Page/..." which obviously won't be correct...